opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -484,6 +486,32 @@ namespace Opm {
             return istlSolver().iterations();
         }
 
+        /// Zero out off-diagonal blocks on rows corresponding to overlap cells
+        /// Diagonal blocks on ovelap rows are set to diag(1e100).
+        void makeOverlapRowsInvalid(Mat& ebosJacIgnoreOverlap) const
+        {	  	  
+            //value to set on diagonal
+            MatrixBlockType diag_block(0.0);
+            for (int ii=0;ii<numEq;++ii)	    
```

Comment:
Please add spaces around binary operators (`=`, `<`) on this line and lines 496, 503, 510, space after semicolon. Always use braces for blocks even if they are just one line. Prefer naming counting variables something reasonable (I think `eq` rather than `ii` would be nice here).